src/ngraph/op/fused/lstm_sequence.hpp

```diff
@@ -66,7 +60,7 @@ namespace ngraph
                                       const Output<Node>& B,
                                       const Output<Node>& P,
                                       const std::int64_t hidden_size,
-                                      const direction lstm_direction,
+                                      const RecurrentSequenceDirection lstm_direction,
```

Comment:
Why do you need an API breaking change?

Reply:
Add a `using direction=RecurrentSequenceDirection` to the class so it won't be API-breaking.

Reply:
I need some enum to use for ONNX's RNN and GRU direction mode. Added using statement.